lib/v2/udn/breaking-news.js

```diff
@@ -0,0 +1,66 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+    const link = `/news/breaknews/1/${id}#breaknews`;
+    const name = await getLinkName(ctx, link);
+    const rootUrl = 'https://udn.com/';
+    const response = await got(`${rootUrl}/api/more?page=1&channelId=1&cate_id=${id}&type=breaknews`);
+    const items = await Promise.all(
+        response.data.lists.map(async (item) => {
+            const link = item.titleLink.indexOf('http://') === 0 || item.titleLink.indexOf('https://') === 0 ? item.titleLink : `${rootUrl}${item.titleLink}`;
```

Comment:
Consider removing the `from` search params from url

Reply:
There is no `from` search params

Reply:
![image](https://user-images.githubusercontent.com/11386903/187430289-de80a70a-01ea-473b-b364-df83f5347699.png)